rest-api-spec/src/main/resources/rest-api-spec/api/engine.list.json

```diff
@@ -0,0 +1,43 @@
+{
+  "engine.list": {
+    "documentation": {
+      "url": "http://todo.com/tbd",
+      "description": "Returns the existing engines."
+    },
+    "stability": "experimental",
+    "visibility": "feature_flag",
+    "feature_flag": "xpack.ent-search.enabled",
+    "headers": {
+      "accept": [
+        "application/json"
+      ],
+      "content_type": [
+        "application/json"
+      ]
+    },
+    "url": {
+      "paths": [
+        {
+          "path": "/_engine",
+          "methods": [
+            "GET"
+          ]
+        }
+      ]
+    },
+    "params": {
+      "q": {
+        "type": "string",
+        "description": "Query in the Lucene query string syntax"
+      },
+      "from": {
+        "type": "number",
```

Comment:
Should `from` be an `int` and not a `number`?

Reply:
Right! Thanks for spotting this 👍